plugins/autoclean.c

```diff
@@ -324,6 +324,14 @@ static struct command_result *listforwards_done(struct command *cmd,
 			continue;
 		}
 
+		/* Check if we have a resolved_time, before making a
+		 * decision on it. This is possible in older nodes
+		 * that predate our annotations for forwards.*/
+		if (json_get_member(buf, t, "resolved_time") == NULL) {
```

Comment:
```suggestion
		if (!json_get_member(buf, t, "resolved_time")) {
```

There is any reason that I'm ignoring to use of this shortcut?

Reply:
I personally prefer being very explicit in these cases, since I often end up not seeing the `!` in front when reading code :-)